Documentation/Parameters/Parameters.md

```diff
@@ -85,6 +85,22 @@ format | result
 
 *Note:* If the parameter value is not valid json, the result of the format will be an empty value.
 
+## Formatting parameters
```

Comment:
for this formatting, i thought we had a big table of the value, formatter and result somewhere?  when possible, also show the input/output as text instead of images

Reply:
All parameters has their own tables. So I added a new file with the new format specifiers and added link to this new file from all other types.